src/cli/graph.js

```diff
@@ -92,6 +97,12 @@ const graphCommand: Command = async (args, std) => {
     const dirContext = pluginDirectoryContext(baseDir, name);
     const weightedGraph = await plugin.graph(dirContext, rd, taskReporter);
 
+    const identities = await plugin.identities(dirContext, taskReporter);
+    for (const identityProposal of identities) {
+      ensureIdentityExists(ledger, identityProposal);
+    }
+    taskReporter.finish(task);
```

Comment:
I notice we change when `task` (which should really be called `generateGraphTask`-- mind renaming?) gets resolved. I think the new semantics might be better? but I want to confirm this is intentional.

Reply:
Correct, this is intentional. I think it's better this way too. More atomic and less nesting. I'll update the variable name.